MarkupsToModel/qSlicerMarkupsToModelModuleWidget.cxx

```diff
@@ -275,391 +245,246 @@ void qSlicerMarkupsToModelModuleWidget::exit()
 void qSlicerMarkupsToModelModuleWidget::setMRMLScene( vtkMRMLScene* scene )
 {
   Q_D( qSlicerMarkupsToModelModuleWidget );
-  qCritical( "PERRAS" );
   this->Superclass::setMRMLScene( scene );
   qvtkReconnect( d->logic(), scene, vtkMRMLScene::EndImportEvent, this, SLOT(onSceneImportedEvent()) );
 }
 
-
 //-----------------------------------------------------------------------------
 void qSlicerMarkupsToModelModuleWidget::onSceneImportedEvent()
 {
   Q_D( qSlicerMarkupsToModelModuleWidget );
-  qCritical( "PERRAS" );
   this->enter();
   this->updateFromMRMLNode();
-  //this->updateWidget();
 }
 
 //-----------------------------------------------------------------------------
-void qSlicerMarkupsToModelModuleWidget::updateWidget()
+void qSlicerMarkupsToModelModuleWidget::updateToMRMLNode()
```

Comment:
I know that would be slightly better for performance. I don't know that it will be a noticeable improvement though (I don't notice any delay in the current implementation), and it makes the code harder to work with (there are a lot of widgets, which == a lot of new function calls, which == many more lines of code to keep track of). I would prefer to leave it as-is, because right now it's easy to see how everything is updated, all in one place.